tests/reconfig/results/1.3.0/1.3.0.md

```diff
@@ -0,0 +1,110 @@
+| 1           | 150          | 5                 | 4457                                 | 40%      | 80%       | 80%       | 80%        | 100%       |
+| 2           | 30           | 371               | 59.5                                 | 99.7%    | 100%      | 100%      | 100%       | 100%       |
+| 2           | 150          | 1742              | 93.5                                 | 92.9%    | 99.99%    | 100%      | 100%       | 100%       |
+| 3           | 30           | 370               | 43.9                                 | 99.85%   | 99.85%    | 100%      | 100%       | 100%       |
+| 3           | 150          | 1810              | 44.8                                 | 99.99%   | 99.99%    | 99.99%    | 100%       | 100%       |
+
+#### Plus
+
+| Test number | NumResources | Event Batch Total | Event Batch Processing avg time (ms) | <= 500ms | <= 1000ms | <= 5000ms | <= 10000ms | <= 30000ms   |
+|-------------|--------------|-------------------|--------------------------------------|----------|-----------|-----------|------------|--------------|
+| 1           | 30           | 6                 | 84                                   | 100%     | 100%      | 100%      | 100%       | 100%         |
+| 1           | 150          | 5                 | 4544.3                               | 40%      | 80%       | 80%       | 80%        | 100%         |
+| 2           | 30           | 370               | 59.1                                 | 100%     | 100%      | 100%      | 100%       | 100%         |
+| 2           | 150          | 1747              | 93.2                                 | 94.1%    | 99.99%    | 100%      | 100%       | 100%         |
+| 3           | 30           | 370               | 41.33                                | 99.99%   | 99.99%    | 100%      | 100%       | 100%         |
+| 3           | 150          | 1809              | 44.88                                | 99.99%   | 99.99%    | 99.99%    | 99.99%     | 100%         |
+
+## NumResources to Total Resources
+
+| NumResources | Gateways | Secrets | ReferenceGrants | Namespaces | application Pods | application Services | HTTPRoutes | Total Resources |
+|--------------|----------|---------|-----------------|------------|------------------|----------------------|------------|-----------------|
+| x            | 1        | 1       | 1               | x+1        | 2x               | 2x                   | 3x         | <total>         |
+| 30           | 1        | 1       | 1               | 31         | 60               | 60                   | 90         | 244             |
+| 150          | 1        | 1       | 1               | 151        | 300              | 300                  | 450        | 1204            |
+
+## Observations
+
+1. Reload time and time to ready have increased in 150 resource tests. This is probably due, in part, to the fix of https://github.com/nginxinc/nginx-gateway-fabric/issues/1107 causing the prior
```

Comment:
Is this the right bug? I thought the bug was with the test code, not the httpmatches implementation.

Reply:
Hmm I see these notes in the 1.2 run

> Note: Only 2x HTTPRoutes attach to the Gateway because the parentRef name in the `cafe-tls-redirect` HTTPRoute is incorrect. This has been fixed, but until https://github.com/nginxinc/nginx-gateway-fabric/issues/1107 is fixed we can't actually run the test successfully.

> ## Future Improvements

> Fix https://github.com/nginxinc/nginx-gateway-fabric/issues/1107 to allow for 150 resource tests to properly run.

Reply:
I see. It's all coming together now. The bug seems unrelated to the test if you just read the description.